Cargo.toml

```diff
@@ -2,6 +2,8 @@
 resolver = "2"
 
 members = [
+    "velato",
```

Comment:
Obviously it's open for discussion, but I'd be inclined to put this in the `integrations` folder

Reply:
To me, integration implies glue code for a third party crate which makes sense for svg but not so much for this. I’ll rename the directory to _crates_ and move both there as that seems to be standard practice for this sort of thing.

Reply:
`crates` to me suggests the opposite dependency flow, i.e. that `crates` are dependencies of the top level crate, not the other way around.

I chose the name integrations for this directory as that maintains the feeling of the crates being dependent on the top level crate. (I'd also claim that this is an integration with `lottie`)

Maybe there's a third name which captures that?